configure.ac

```diff
@@ -69,7 +69,7 @@ AS_IF([ test -n "$CFLAG_VISIBILITY" ], [
        AM_CFLAGS="$AM_CFLAGS $CFLAG_VISIBILITY"
        ])
 
-WOLFSSL_BUILD_DATE=$(date -R)
+WOLFSSL_BUILD_DATE=$(env LC_TIME=C date +"%a, %d %b %Y %T %z")
```

Comment:
We aren't using `env` elsewhere -- can't we just do this with
```
WOLFSSL_BUILD_DATE=$(LC_TIME=C date +"%a, %d %b %Y %T %z")
```
?

Reply:
Thanks, I've confirmed that setting LC_TIME=C without preceding ```env``` works on old system (FreeBSD 4.11 (2005)). I've pushed updated one.